config/initializers/redis.rb

```diff
@@ -9,14 +9,16 @@
   ENV['REDIS_URL'] = "redis://#{password.blank? ? '' : ":#{password}@"}#{host}:#{port}/#{db}"
 end
 
-Redis.current = Redis.new(
+namespace = ENV.fetch('REDIS_NAMESPACE') { '' }
```

Comment:
ah shit trash will there be a : in there even with '' namespace? if so that's a bug and i will fix it. my goal was for these changes to do nothing at all if people don't have a namespace set

your timelines would fill up again but they would be empty immediately after the update because as you say, the streaming service would be looking in a new place

the rails side will add the stuff to the new place so things will return to normal as new toots arrived, but having everything disappear would be jarring and bad i think